### ISSUE TITLE
[BUG] Song loops at arbitrary timestamps

### ISSUE THREAD

**Issue by alx-alexpark** [snuffyDev/Beatbump#134]
How to reproduce:
- Play a song
- Wait until timestamp reaches halfway, or some other timestamp

Expected behavior: Song plays to the end.
Actual behavior: Song loops back to start at that arbitrary timestamp.

**Comment by IronMaltese** [comment]
What operating system and browser are you using and also have you changed any settings

**Comment by alx-alexpark** [comment]
I have not changed any settings. I am using Brave on macOS.

**Comment by snuffyDev** [comment]
Yeah I noticed this on Mac as well, debugging it currently. I’ll make a reply once I figure it out

**Comment by alx-alexpark** [comment]
Sounds great! I love beatbump, just hope that some of the bugs get patched sometime.

**Comment by snuffyDev** [comment]
Would you happen to have a reproduction method?

What version of Mac OS and Safari are you running? I have a VM and remote device testing setup, so I can hopefully get to the bottom of this

**Comment by alx-alexpark** [comment]
macOS 13.1 (22C65)
Not using safari, using Brave 1.46.144 Chromium: 108.0.5359.128 (Official Build) (arm64)

**Comment by alx-alexpark** [comment]
Sorry for the late response

**Comment by snuffyDev** [comment]
That’s okay! I think I figured out what the problem may be, still working on a fix

**Comment by snuffyDev** [comment]
I think it might be fixed now? Hopefully it is!
